arex-instrumentation-api/src/main/java/io/arex/inst/runtime/util/TypeUtil.java

```diff
@@ -119,18 +119,9 @@ private static String genericTypeToString(Object result) {
         for (int i = 0; i < typeParameters.length; i++) {
             final String typeName = typeParameters[i].getTypeName();
             String cacheKey = rawClassName + typeName;
-            Field field = GENERIC_FIELD_CACHE.get(cacheKey);
-            if (field == null) {
-                for (Field declaredField : rawClass.getDeclaredFields()) {
-                    // java.util.List<T> contains T
-                    if (declaredField.getGenericType().getTypeName().contains(typeName)) {
-                        declaredField.setAccessible(true);
-                        GENERIC_FIELD_CACHE.put(cacheKey, declaredField);
-                        field = declaredField;
-                        break;
-                    }
-                }
-            }
+            Field field = GENERIC_FIELD_CACHE.computeIfAbsent(cacheKey, genericField ->
+                    getGenericFieldFromClass(rawClass, typeName));
+            field.setAccessible(true);
```

Comment:
There may be NPE here because `getGenericFieldFromClass` returns null.
Recommended: put `field.setAccessible(true);` before return in the `getGenericFieldFromClass` method